packages/ui/src/components/organisms/DynamicForm/components/RSJVInputAdaters/DateInputAdater/DateInputAdater.tsx

```diff
@@ -36,9 +37,19 @@ export const DateInputAdater: RJSFInputAdapter<string | null> = ({
     [onChange],
   );
 
+  const datePickerParams = useMemo(() => {
+    const params: DatePickerParams = {
+      disableFuture: uiSchema?.disableFutureDate,
+      disablePast: uiSchema?.disablePastDate,
+    };
+
+    return params;
```

Comment:
Please inline return or change to bracket-less component that returns an object

Reply:
Thinking about it, you can inline it into the return statement of the entire component and it'll be fine, `useMemo` isn't crucial here